packages/cli/package.json

```diff
@@ -95,6 +95,7 @@
     "@stacks/stacking": "^2.0.0-beta.1",
     "@stacks/storage": "^2.0.0-beta.1",
     "@stacks/transactions": "^2.0.0-beta.1",
+    "@stacks/wallet-sdk": "^1.0.0-wallet-sdk.4",
```

Comment:
Looks like this PR is blocked on https://github.com/blockstack/stacks.js/pull/897 ?